SPECS/application-gateway-kubernetes-ingress/application-gateway-kubernetes-ingress.spec

```diff
@@ -1,4 +1,7 @@
 %global debug_package %{nil}
+# The commit corresponds to the project release version
```

Comment:
Thank you for adding that. Please also say what this is for. I'm guessing it has to be updated for each version update, so that's something we should definitely mention.